johnwayodi/sm-server
# #161695595 Update API Documentation

#### What does this PR do?
Update API documentation to include all added endpoints
#### Description of Task to be completed?
1. Create `.yml` files for all the resource methods.
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api-v2.git`
2. Ensure you are in the cloned repo, **sm-api-v2** by default.
3. Create a virtual environment using virtualenv and `pip install -r requirements.txt`
4. `git checkout -b ch-update-apidocs-161695595 origin/ch-update-apidocs-161695595`
5. Run **_pytest -vv --cov=storemanager/api/v2_** in terminal to run the tests and get coverage on the application.
6. Run the flask application and visit `localhost:5000/apidocs` to view and use the docs.
#### Any background context you want to provide?
No background context
#### What are the relevant pivotal tracker stories?
[#161695595](https://www.pivotaltracker.com/story/show/161695595)
#### Questions:
How can I arrange the endpoints so that `auth/` comes first then the rest?
